website/docs/reference/release-templates.mdx

```diff
@@ -0,0 +1,104 @@
+---
+title: Release templates
+---
+
+:::note Availability
+
+**Plan**: [Enterprise](https://www.getunleash.io/pricing) cloud-hosted | **Version**: `6.8+` in [BETA](/availability#beta-features).
+
+:::
+
+<details>
+  <summary>Share your feedback</summary>
+
+  We're currently gathering feedback for release templates and release plans.
+  
+  To provide feedback using the Admin UI, go to **Configure > Release templates**, and click **Share your feedback**.
+
+  Future iterations of the feature may include:
+    - An overview of ongoing releases and currently running milestones
+    - Automated transitions between different milestones
```

Comment:
I suspect this list of potential future improvements was discussed with Product leadership, did you also happen to discuss cross environment templates?

Reply:
Anniken hasn't mentioned that, but I'll ask. thanks!

Reply:
Anniken confirm that we shouldn't mention this yet